src/dialogs/more-info/controls/more-info-light.ts

```diff
@@ -141,6 +145,7 @@ class MoreInfoLight extends LitElement {
                 ${supportsColorTemp || supportsColor
                   ? html`
                       <md-outlined-icon-button
+                        class="color-mode"
```

Comment:
Can we make this use `supportsColorTemp` & `supportsColor` and only show the color wheel when `supportsColor` and otherwise show a gradient of color temp?